packages/soundxyz/src/Soundxyz.ts

```diff
@@ -137,27 +137,26 @@ export const simulateMint = async (
 export const getProjectFees = async (
   mint: MintActionParams,
 ): Promise<bigint> => {
-  const { chainId, contractAddress } = mint
+  const { chainId, contractAddress, tokenId } = mint
 
+  const tier = tokenId ?? 0
   const client = createPublicClient({
     chain: chainIdToViemChain(chainId),
     transport: http(),
   })
 
-  const mintInfoList = (await client.readContract({
+  const nextSchedule = (await client.readContract({
     address: SUPERMINTER_V2,
     abi: MINT_INFO_LIST_ABI,
-    functionName: 'mintInfoList',
-    args: [contractAddress],
-  })) as MintInfoList
```

Comment:
Since we no longer use `MintInfoList` can you remove the `MintInfo` and `MintInfo` types?

Or can just delete `questdk-plugins/packages/soundxyz/src/types.ts`